data/yam/agama/auto/alp_leap16.sh

```diff
@@ -0,0 +1,8 @@
+set -ex
```

Comment:
should be called `auto_leap16` is not related with alp.

Reply:
thanks, I update the file name.